pkgs/top-level/python-packages.nix

```
@@ -10309,7 +10310,7 @@ self: super: with self; {
   openapi3 = callPackage ../development/python-modules/openapi3 { };
 
   openbabel-bindings = callPackage ../development/python-modules/openbabel-bindings {
-    openbabel = callPackage ../development/libraries/openbabel { inherit (self) python; };
+    openbabel = pkgs.openbabel.override { python3 = self.python; };
```

```suggestion
    openbabel = pkgs.openbabel.override { python3 = python; };
```

The `self` here should be redundant.